app/assets/javascripts/application/tools/congress_message.js

```diff
@@ -247,6 +247,18 @@ $(document).on("ready", function() {
             },
             error: function() {}
           });
+        },
+        onDefunctLegislator: function(bioguide, contact_url) {
+          var notice = $("<p>").addClass("defunct-notice")
```

Comment:
Do we support blank contact_urls?  I thought that was allowed, since they're optional in the rake tasks.